demo/simpleBenchmark.R

```diff
@@ -1,6 +1,14 @@
+#!/usr/bin/env Rscript
 
+if (!require("microbenchmark")) install.packages("microbenchmark")
```

Comment:
Or simply a `stopifnot()` which now has customizable errors. 

Auto-installation is no panacea either as not every user will have a compiler.